content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -339,32 +341,22 @@ const result = prisma.$executeRawUnsafe(
 $executeRawUnsafe<T = unknown>(query: string, ...values: any[]): PrismaPromise<number>;
 ```
 
-### Using <inlinecode>improvedQueryRaw</inlinecode> (Preview)
+### Raw query type mapping
 
-In versions `3.14.0` and later, improvements to the raw query API are available:
+<Admonition type="warning">
 
-- Values returned from raw queries are mapped to the corresponding JavaScript type. For more information, see [Raw query type mapping](#raw-query-type-mapping)
-- Typecasting issues in PostgreSQL are fixed. For more information, see [PostgreSQL typecasting fixes](#postgresql-typecasting-fixes)
+This feature is currently not available for SQLite.
```

Comment:
Apologies if it wasn't made clear, but it is now also available for SQLite on Prisma 4

Reply:
Great, thanks, I'll add a note about this